### ISSUE TITLE
Out-of-Domain examples

### ISSUE THREAD

**Issue by bkozyrskiy** [fdbtrs/ElasticFace#15]
Can you elaborate on
`index = torch.where(label != -1)[0]` 
in utils/losses.py? This line is present for various losses. 
Do I understand correctly that this is a mechanism for contre-examples, i.e. images that are not faces or faces not subjected for recognition?
